tests/tests.toml

```diff
@@ -497,6 +497,10 @@ asr = true
 filename = "../integration_tests/global_syms_01.py"
 asr = true
 
+[[test]]
+filename = "../integration_tests/test_const_dict.py"
+asr = true
+
```

Comment:
I think we need not test this here. Testing it as an `integration_test` is sufficient.

Reply:
After you remove this from here, please also remove the generated reference files.